apps/google-analytics-4/frontend/src/components/main-app/AnalyticsApp/AnalyticsApp.tsx

```diff
@@ -63,8 +52,9 @@ const AnalyticsApp = (props: Props) => {
       setLoading(false);
     }
 
-    fetchRunReportData();
-  }, [api, reportRequestParams]);
+    if (reportSlug && propertyId && !isContentTypeWarning) fetchRunReportData();
```

Comment:
not calling this endpoint if we do not have required slug or propertyId params, and if there is a content related warning.